src/components/DataTable/DataTable.scss

```diff
@@ -162,7 +162,7 @@ $breakpoint: 768px;
 
 .Footer {
   padding: spacing();
-  background: color('sky', 'light');
-  color: color('ink', 'lighter');
+  background: var(--p-surface-foreground-subdued, color('sky', 'lighter'));
```

Comment:
I prefer not be be changing fallback values as part of this PR

Reply:
```suggestion
  background: var(--p-surface-foreground-subdued, color('ink', 'lighter'));
```